x-pack/plugins/index_lifecycle_management/public/application/sections/edit_policy/components/phases/shared_fields/data_tier_allocation_field/components/will_use_fallback_tier_notice.tsx

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+import React, { FunctionComponent } from 'react';
+import { EuiCallOut } from '@elastic/eui';
+
+import { PhaseWithAllocation, DataTierRole } from '../../../../../../../../../common/types';
+
+const nodeRoleToFallbackTierMap: Partial<Record<DataTierRole, string>> = {
+  data_hot: i18n.translate('xpack.indexLifecycleMgmt.editPolicy.dataTierHotLabel', {
+    defaultMessage: 'hot',
+  }),
+  data_warm: i18n.translate('xpack.indexLifecycleMgmt.editPolicy.dataTierWarmLabel', {
+    defaultMessage: 'warm',
+  }),
+};
+
+const i18nTexts = {
+  warm: {
+    title: i18n.translate('xpack.indexLifecycleMgmt.warmPhase.dataTier.willUseFallbackTierTitle', {
+      defaultMessage: 'No nodes assigned to the warm tier',
+    }),
+    body: (nodeRole: DataTierRole) =>
+      i18n.translate('xpack.indexLifecycleMgmt.warmPhase.dataTier.willUseFallbackTierDescription', {
+        defaultMessage:
+          'This policy will move data in the warm phase to {tier} tier nodes instead.',
+        values: { tier: nodeRoleToFallbackTierMap[nodeRole] },
+      }),
+  },
+  cold: {
+    title: i18n.translate('xpack.indexLifecycleMgmt.coldPhase.dataTier.willUseFallbackTierTitle', {
+      defaultMessage: 'No nodes assigned to the cold tier',
+    }),
+    body: (nodeRole: DataTierRole) =>
+      i18n.translate('xpack.indexLifecycleMgmt.coldPhase.dataTier.willUseFallbackTierDescription', {
+        defaultMessage:
+          'This policy will move data in the cold phase to {tier} tier nodes instead.',
+        values: { tier: nodeRoleToFallbackTierMap[nodeRole] },
+      }),
+  },
+};
+
+const customizeWithNodeAttributeDescription = i18n.translate(
+  'xpack.indexLifecycleMgmt.editPolicy.customizeWithNodeAttributeDescription',
+  {
+    defaultMessage:
+      'Define custom node attributes in elasticsearch.yml to allocate indices based on these attributes.',
```

Comment:
It's not clear why this is relevant in the context of the fallback tiers?

Reply:
This ties back to https://github.com/elastic/kibana/pull/96111#discussion_r619697061. In the situation where this message is shown to the user, they've selected "Custom" allocation but there are no custom node attributes available. If they've chosen "Custom" I expect their intention would be to use custom node attributes to control allocation, otherwise they should use the "Recommended (data tiers)" option.